code/__HELPERS/roundend.dm

```diff
@@ -129,6 +129,46 @@
 					var/client/C = GLOB.directory[ckey(A.owner.key)]
 					if (C)
 						greentexters |= C
+					switch(A.type)
```

Comment:
the ideal way to implement this would be to make proc in antagonist giving the medals instead of hardchecking types.

Reply:
Proc implemented